abletonosc/clip.py

```diff
@@ -70,27 +70,49 @@ def clip_callback(params: Tuple[Any]) -> Tuple:
             "remove_notes_by_id"
         ]
         properties_r = [
+            ##"end_time" ## doesn't seem to do anything, can't test
```

Comment:
This property seems to work fine (tested on MIDI clips, it indicates the end of the loop brace) so this line can be uncommented